src/components/KeyboardShortcutsModal/KeyboardShortcutsModal.js

```diff
@@ -0,0 +1,85 @@
+import _ from 'lodash';
```

Comment:
@elsenhans could you please import just what you need from lodash?

`import { upperFirst } from lodash`;

should work do it here.

Reply:
I changed _lodash_ import to _upperFirst_ import.